sig-advocacy-and-outreach/summer-ospp/TODO-LIST.md

```diff
@@ -3,7 +3,7 @@
 > 1. 可以直接在 GitHub 上提交完善信息
 > 2. 可以直接在 HackMD 上补充完善信息，由管理员统一提交至 GitHub.
 
-- [ ] KubeSphere Jenkins Client Refactor @Rick 
+- [x] KubeSphere Jenkins Client Refactor @Rick 
```

Comment:
The GitHub ID should be [linuxsuren](https://github.com/linuxsuren) instead of `@Rick`